requirements-dev.txt

```diff
@@ -18,5 +18,5 @@ mypy>=0.780
 mypy-extensions>=0.4.3
 nbsphinx
 qiskit_sphinx_theme>=1.10.*
-networkx>=2.2,<2.6
+networkx>=2.6.3
```

Comment:
Do we even need this here? I searched and this is the only hit I found for networkx - so since we don;t use this in finance, as far as I can see, can we not just rely on it being pulled in via the requirements of qiskit optimzation when that dependent is installed?

Reply:
I just saw your comment. No we don't need it. I removed it from the requirements.

Reply:
I saw the same requirement in ML but it doesn't use networkx anywhere